addons/hr_contract/wizard/hr_plan_wizard.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class HrPlanWizard(models.TransientModel):
+    _inherit = 'hr.plan.wizard'
+
+    @api.model
+    def default_get(self, field_list=None):
+        result = super(HrPlanWizard, self).default_get(field_list)
+        selected_employees = self.env['hr.employee'].browse(self.env.context['active_ids'])
+        start_dates = [d for d in selected_employees.mapped('first_contract_date') if d]
```

Comment:
Same as - but you can use whatever you prefer :-)

```suggestion
        start_dates = selected_employees.filtered('first_contract_date').mapped('first_contract_date')
```